### ISSUE TITLE
Error creating snapshot

### ISSUE THREAD

**Issue by geoffroya** [elastic/curator#607]
Hello

I want to use curator to create snapshot, but when running the command I get the following output:
```
$ curator snapshot --repository tests-results --prefix test --include_global_state False indices --time-unit days --older-than 2 --timestring '%Y.%m.%d'
Traceback (most recent call last):
  File "/usr/local/bin/curator", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/curator/curator.py", line 5, in main
    cli( obj={ "filters": [] } )
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1057, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/curator/cli/snapshot.py", line 46, in snapshot
    if any(c.isupper() for c in name):
TypeError: 'NoneType' object is not iterable
```

ES 2.3.1
Curator 3.5.0

The repository has been created with the following command:
```
curl -XPUT localhost:9200/_snapshot/test-results -d '{
  "type": "fs",
  "settings": {
    "location": "/mnt/nfs4/elastic/test-results",
    "compress": "true"
  }
}'
```

My cluster has the following indices:

```
$  curl localhost:9200/_cat/indices
green open simulators-2016.04.25       2 0   2903 0   2.2mb   2.2mb
green open simulators-2016.04.26       2 0  83405 0  27.5mb  27.5mb
green open logs-2016.04.19             2 0   3992 0     2mb     2mb
green open logs-2016.04.25             2 0 125999 0  98.9mb  98.9mb
green open logs-2016.04.26             2 0   6152 0  12.5mb  12.5mb
green open grokparsefailure-2016.04.25 2 0    280 0 170.7kb 170.7kb
green open grokparsefailure-2016.04.26 2 0   1227 0 166.6kb 166.6kb
green open .kibana                     1 0     24 2  57.2kb  57.2kb
green open logs-2016.04.23             2 0   8626 0   6.8mb   6.8mb
green open logs-2016.04.24             2 0   9347 0   8.5mb   8.5mb
green open logs-2016.04.21             2 0  41519 0  60.1mb  60.1mb
green open logs-2016.04.22             2 0 178914 0 113.9mb 113.9mb
green open simulators-2016.04.21       2 0  11618 0   7.2mb   7.2mb
green open simulators-2016.04.22       2 0   7371 0   5.1mb   5.1mb
green open logs-2016.04.20             2 0  25559 0  32.2mb  32.2mb
```

Any thoughts?

Best regards

Geoff.

**Comment by untergeek** [comment]
This is a duplicate of #584, which issue has been fixed in version 3.5.1.

Since you're on 3.5.0, an upgrade should fix that.

**Comment by geoffroya** [comment]
Thanks @untergeek
